docs/pt-br/gettingstarted.md

```diff
@@ -19,15 +19,15 @@ dependencies:
   bloc: ^3.0.0
 ```
 
-Para uma aplicação [Flutter](https://flutter.io), também é necessário adicionar o package flutter_bloc como uma dependência no `pubspec.yaml`.
+Para uma aplicação [Flutter](https://flutter.dev), também é necessário adicionar o package flutter_bloc como uma dependência no `pubspec.yaml`.
 
 ```yaml
 dependencies:
   bloc: ^3.0.0
   flutter_bloc: ^3.1.0
 ```
 
-Para uma aplicação [AngularDart](https://webdev.dartlang.org/angular), também é necessário adicionar o package angular_bloc como uma dependência no `pubspec.yaml`.
+Para uma aplicação [AngularDart](https://angulardart.dev), também é necessário adicionar o package angular_bloc como uma dependência no `pubspec.yaml`.
```

Comment:
```suggestion
Para uma aplicação [AngularDart](https://angulardart.dev/), também é necessário adicionar o package angular_bloc como uma dependência no `pubspec.yaml`.
```

Reply:
why is the trailing slash necessary?

Reply:
https://github.com/felangel/bloc/pull/813#discussion_r374004174